pkgs/applications/audio/roc-toolkit/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, scons
+, python
+, ragel
+, gengetopt
+, libuv
+, libpulseaudio
+, sox
+}:
+
+stdenv.mkDerivation {
+  name = "roc-toolkit";
+
+  nativeBuildInputs = [ scons python ragel gengetopt ];
+  buildInputs = [ libuv libpulseaudio sox ];
+
+  src = fetchFromGitHub {
+    owner = "roc-streaming";
+    repo = "roc-toolkit";
+    rev = "c89687330bfce6f4dce79826f7a235b581f2b49d";
```

Comment:
Why not use https://github.com/roc-streaming/roc-toolkit/releases/tag/v0.1.5 ?

Reply:
> I have not had success compiling the latest tagged release which is v0.1.5 from
April 2020. Latest master, with last commit dated Jun 15, compiles
successfully however.